esmvaltool/config-references.yml

```diff
@@ -74,6 +74,10 @@ authors:
     email: klaus.zimmermann@smhi.se
     orcid: https://orcid.org/0000-0003-3994-2057
   # Development team
+  abdollahi_banafsheh:
+    name: Abdollahi, Banafsheh
+    institute: University of Delft,Netherlands
```

Comment:
Delft University of Technology, The Netherlands

Reply:
Affiliation modified.